src/Uno.UWP/UI/ColorOffset.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Windows.UI
+{
+	/// <summary>
+	/// Mapping of a <see cref="Color"/> which permits negative values, which is convenient for calculations during animation.
+	/// </summary>
+	internal struct ColorOffset
+	{
+		public int A { get; set; }
+
+		public int B { get; set; }
+
+		public int G { get; set; }
+
+		public int R { get; set; }
+
+
+
+		private ColorOffset(int a, int r, int g, int b)
+		{
+			A = a;
+			R = r;
+			G = g;
+			B = b;
+		}
+
+		public static ColorOffset FromArgb(int a, int r, int g, int b) => new ColorOffset(a, r, g, b);
+
+		public static explicit operator Color(ColorOffset colorOffset) => Color.FromArgb((byte)colorOffset.A, (byte)colorOffset.R, (byte)colorOffset.G, (byte)colorOffset.B);
+
+		public static explicit operator ColorOffset(Color color) => FromArgb(color.A, color.R, color.G, color.B);
+
+		public static ColorOffset operator -(ColorOffset minuend, ColorOffset subtrahend)
+			=> FromArgb(minuend.A - subtrahend.A, minuend.R - subtrahend.R, minuend.G - subtrahend.G, minuend.B - subtrahend.B);
+
+		public static ColorOffset operator +(ColorOffset first, ColorOffset second)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Implement alternative method 'Add' for the operator '+'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5301553)